web/src/components/localization-select/localization-select-complex.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
A lot of this new component is lifted directly from the [`useSelect` docks for downshift](https://github.com/downshift-js/downshift/tree/master/src/hooks/useSelect).